src/main/java/org/terasology/smithing/system/CharcoalPitAuthoritySystem.java

```diff
@@ -102,6 +121,14 @@ public void startBurningCharcoal(ProduceCharcoalRequest event, EntityRef entity,
         }
     }
 
+    /*
+    * Adds the produced charcoal to the players inventory
```

Comment:
Does it add the charcoal to the _players_ inventory? Wouldn't it be put in the charcoal pit's inventory? I might be totally wrong here, I haven't checked the code, but it would seem weird for the pit to dump its output into a player inventory :-)